test/mlir/onnx/onnx_shape_inference.mlir

```diff
@@ -2289,3 +2289,36 @@ func @test_category_mapper_int64 (%arg0: tensor<20x1xi64>) -> tensor<*x!onnx.Str
   // CHECK: [[RES:%.+]] = "onnx.CategoryMapper"(%arg0) {cats_int64s = [1, 2, 3], cats_strings = ["cat", "dog", "human"], default_string = "unclassified" : !onnx.String} : (tensor<20x1xi64>) -> tensor<20x1x!onnx.String>
   // CHECK: return [[RES]] : tensor<20x1x!onnx.String>
 }
+
+// -----
```

Comment:
Could you please add an empty line after `// -----`? It is condition for `split-input-file` to work.